pvanalytics/util/_fit.py

```diff
@@ -4,9 +4,10 @@
 
 
 def _quadratic(xs, ys):
-    # fit a quadratic function of `xs` to the data in `ys`
-    coefficients = np.polyfit(xs, ys, 2)
-    return np.poly1d(coefficients)
+    # fit a quadratic function of `xs` to the data in `ys`  
+    coefficients = list(np.polynomial.Polynomial.fit(xs, ys, 2).convert())
```

Comment:
I think if you use `polynomial.polyfit` and `flipud` this would be simpler.

Reply:
![image](https://user-images.githubusercontent.com/70228568/148136567-d0b5cfec-5a65-46b1-b971-bc5b5cb54787.png)

Tried it out and verified same results--thank you for the suggestion @cwhanse !